admin/src/components/Help/OnboardingTour.tsx

```
@@ -0,0 +1,238 @@
+						className="wppo-tour-popup__close"
+						onClick={ skipTour }
+						aria-label="Close tour"
+					>
+						<span className="dashicons dashicons-no-alt"></span>
+					</button>
+				</div>
+
+				<div className="wppo-tour-popup__content">
+					<div
+						className="wppo-tour-popup__text"
+						dangerouslySetInnerHTML={ { __html: step.content } }
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)